.travis.yml

```diff
@@ -4,4 +4,12 @@ services:
     - docker
   
 script:
-    - ./test-build.sh
+    - ./test-build.sh $NODE_VERSION
```

Comment:
Yeah, that would be useful and something we could work on as a separate issue. 

 It would also be great of the `update.sh` script could handle major/minor version changes so that we didn't have to update the respective directories (like when going from `4.6/` --> `4.7/`.  That's probably worth exploring in a separate issue PR.